cluster-commands.go

```diff
@@ -261,9 +263,23 @@ type SRPolicyMapping struct {
 
 // SRSTSCredential - represents an STS credential to be replicated.
 type SRSTSCredential struct {
-	AccessKey    string `json:"accessKey"`
-	SecretKey    string `json:"secretKey"`
-	SessionToken string `json:"sessionToken"`
+	AccessKey           string `json:"accessKey"`
+	SecretKey           string `json:"secretKey"`
+	SessionToken        string `json:"sessionToken"`
+	ParentUser          string `json:"parentUser"`
+	ParentPolicyMapping string `json:"parentPolicyMapping,omitempty"`
+}
+
+// SRIAMUser - represents a regular (IAM) user to be replicated. A nil UserReq
+// implies that a user delete operation should be replicated on the peer cluster.
+type SRIAMUser struct {
+	AccessKey string              `json:"accessKey"`
+	UserReq   *AddOrUpdateUserReq `json:"userReq"`
```

Comment:
```suggestion
	UserReq   *AddOrUpdateUserReq `json:"userReq"`
	DeleteReq bool
```

Explicit key is better than using `nil` for UserReq to perform Delete calls.